src/pages/settings/components/inquiry/InquiryAttachmentSection.tsx

```
@@ -11,20 +12,38 @@ export default function InquiryAttachmentSection({
+  const previews = useMemo(
+    () =>
+      images.slice(0, 5).map((file) => ({
+        file,
+        url: URL.createObjectURL(file),
+      })),
+    [images]
+  );
+
+  useEffect(() => {
+    return () => {
+      previews.forEach((p) => URL.revokeObjectURL(p.url));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)